deploy/roles/spark/defaults/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+spark_tar_file:             "spark-1.5.2-bin-without-hadoop.tgz"
+spark_mirror:               "http://apache.claz.org/spark/spark-1.5.2/spark-1.5.2-bin-without-hadoop.tgz"
```

Comment:
@olivermel it turns out that we do need the version with Hadoop, otherwise sparks expects you to install it separately.  Use the version with hadoop2.6, I tested it and it works fine.  Thanks!